internal/providers/pluginfw/products/library/resource_library_acc_test.go

```diff
@@ -92,6 +94,42 @@ func TestAccLibraryReinstalledIfClusterDeleted(t *testing.T) {
 		})
 }
 
+func TestAccLibraryInstallIfClusterTerminated(t *testing.T) {
+	var clusterId string
+	acceptance.WorkspaceLevel(t,
+		acceptance.Step{
+			Template: commonClusterConfig,
+			Check: func(s *terraform.State) error {
+				clusterId = s.RootModule().Resources["databricks_cluster.this"].Primary.ID
+				return nil
+			},
+		},
+		// If the cluster is Terminated before apply, it should be restarted before installing library.
+		acceptance.Step{
+			PreConfig: func() {
+				// Delete the created cluster
+				w := databricks.Must(databricks.NewWorkspaceClient())
+				getter, err := w.Clusters.Delete(context.Background(), compute.DeleteCluster{
```

Comment:
What a confusing name for this method... @renaudhartert-db maybe we want to revisit this in the next version of the SDK. It should really be "terminate cluster" to align with basically all other terminology.